2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -121,7 +122,8 @@ class CollectorService(
       List(
         RawHeader("P3P", "policyref=\"%s\", CP=\"%s\"".format(config.p3p.policyRef, config.p3p.CP)),
         accessControlAllowOriginHeader(request),
-        `Access-Control-Allow-Credentials`(true)
+        `Access-Control-Allow-Credentials`(true),
+        `Cache-Control`(`no-cache`, `no-store`, `must-revalidate`)
```

Comment:
I'd rather we send this header back just in case we send back the pixel not in all cases

Reply:
Is there a risk if we send it with all responses?

Reply:
I wouldn't think so but it's good practice

Reply:
Done.